{"name": "Проверка создания курьера с логином, который уже есть", "status": "passed", "description": "Проверка, что при создании курьера, который уже есть, возвращается статус 409 и сообщение \"Этот логин уже используется. Попробуйте другой.\"", "start": 1727615343843, "stop": 1727615343984, "uuid": "34849167-eb01-4b23-92f6-5d4dddf88001", "historyId": "84dba8b03acba865e4ab8a4818ff2a8e", "testCaseId": "84dba8b03acba865e4ab8a4818ff2a8e", "fullName": "tests.test_creat_courier.TestCreateCourier#test_create_courier_with_old_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_creat_courier"}, {"name": "subSuite", "value": "TestCreateCourier"}, {"name": "host", "value": "MacBook-Pro--Polanski.local"}, {"name": "thread", "value": "44527-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_creat_courier"}]}